x/evidence/keeper/infraction.go

```diff
@@ -109,6 +113,7 @@ func (k Keeper) HandleEquivocationEvidence(ctx sdk.Context, evidence *types.Equi
 		consAddr,
 		k.slashingKeeper.SlashFractionDoubleSign(ctx),
 		evidence.GetValidatorPower(), distributionHeight,
+		stakingtypes.DoubleSign,
```

Comment:
this doesn't seem needed? the slash amount signals the slash type. I might be missing some context?

Reply:
going through the paths this isn't needed to slash validators, I think we can remove it. Its only needed here https://github.com/cosmos/interchain-security/blob/main/x/ccv/provider/keeper/relay.go#L244